autofs-5.1.3 - be silent about nis domain not set

From: Ian Kent <raven@themaw.net>

When initializing the lookup module for NIS or NISPLUS if the domain
is found to be not set then the service is not configured.

So be silent about the failure.

Signed-off-by: Ian Kent <raven@themaw.net>
---
 CHANGELOG                |    1 +
 modules/lookup_nisplus.c |    1 -
 modules/lookup_yp.c      |    2 --
 3 files changed, 1 insertion(+), 3 deletions(-)

--- autofs-5.0.7.orig/CHANGELOG
+++ autofs-5.0.7/CHANGELOG
@@ -261,6 +261,7 @@
 - fix nisplus lookup init not configured check.
 - make open_lookup() error handling more consistent.
 - be silent about sss library not found.
+- be silent about nis domain not set.
 
 25/07/2012 autofs-5.0.7
 =======================
--- autofs-5.0.7.orig/modules/lookup_nisplus.c
+++ autofs-5.0.7/modules/lookup_nisplus.c
@@ -49,7 +49,6 @@ static int do_init(const char *mapfmt,
 	 */
 	ctxt->domainname = nis_local_directory();
 	if (!ctxt->domainname || !strcmp(ctxt->domainname, "(none).")) {
-		logmsg(MODPREFIX "NIS+ domain not set");
 		ret = 1;
 		goto out;
 	}
--- autofs-5.0.7.orig/modules/lookup_yp.c
+++ autofs-5.0.7/modules/lookup_yp.c
@@ -127,8 +127,6 @@ static int do_init(const char *mapfmt,
 		/* This should, but doesn't, take a const char ** */
 		err = yp_get_default_domain(&domainname);
 		if (err) {
-			logerr(MODPREFIX
-			      "map %s: %s", ctxt->mapname, yperr_string(err));
 			ret = 1;
 			goto out;
 		}
